example/src/App.js

```diff
@@ -52,6 +53,7 @@ const LogoImage = styled("img")`
 
 function App() {
   const [anchorEl, setAnchorEl] = useState(null);
+  const [drawerIsOpen, setDrawerIsOpen] = useState(false);
   const [isRestMode, setIsRestMode] = useState(
```

Comment:
What do you think about defaulting to having the nav drawer open in REST mode? The likely scenario in REST mode is to operate on various files and options and there isn't a clear default -- whatever we choose is likely to be imperfect and the user will likely want to choose a different option.

Perhaps something like

```ts
const [isDrawerOpen, setIsDrawerOpen] = useState(isRestMode);
```

Not sure if we'd need to do `useState(Boolean(isRestMode))`; I don't think we do and I can't articulate why I think we might.